src/lib.rs

```diff
@@ -267,7 +273,11 @@ fn get_runner() -> Result<PathAndArgs> {
         .unwrap_or_else(|| {
             (
                 PathAndArgs::new("wasmtime")
-                    .args(vec!["-S", "preview2", "-S", "common"])
+                    .args(if serve {
+                        vec!["serve"]
+                    } else {
+                        vec!["-S", "preview2", "-S", "cli"]
```

Comment:
Change this back to `vec!["-S", "preview2", "-S", "common"]`? `"common"` vs `"cli"`

I always struggle to find the docs for these `-S` flags. But the CI is failing `error: unknown -S / --wasi option: cli`

Reply:
Ah, good catch. Created an [issue to track](https://github.com/bytecodealliance/cargo-component/issues/306) and change this in the future.